docs/getting-started/overlay.rst

```diff
@@ -223,8 +223,8 @@ the application).
     (myoverlay)$ inveniomanage config set CFG_DATABASE_NAME mysql-database
     (myoverlay)$ inveniomanage config set CFG_DATABASE_USER mysql-user
     # HOST configuration (for redirects, etc.)
-    (myoverlay)$ inveniomanage config set CFG_SITE_URL http://0.0.0.0:4000
-    (myoverlay)$ inveniomanage config set CFG_SITE_SECURE_URL https://0.0.0.0:4000
+    (myoverlay)$ inveniomanage config set CFG_SITE_URL http://localhost:4000
+    (myoverlay)$ inveniomanage config set CFG_SITE_SECURE_URL https://localhost:4000
```

Comment:
this might be confusing ... shall we completely remove it or show `http://invenio.example.com` and `https://invenio.example.com`?